paper/paper.md

```diff
@@ -0,0 +1,281 @@
+
+![\textbf{Creation of the new Lausanne2018 Connectome Parcellation.}
+\label{fig:parc}](Lausanne2018_parcellation_diagram.png)
+
+The parcellation derived from the Desikan-Killiany atlas [@Desikan2006AnInterest] has been
+  made symmetric by projecting the right hemisphere labels to the left hemisphere, matching the
+  boundaries of the projected regions of the left hemisphere to the boundaries of the original regions
+  of the left hemisphere, applying this transformation to the rest of the scales, and saving
+  each parcellation scale in a Freesurfer annotation file.
+After generating the volumetric parcellations from the annotation files, one can now decide or not
+  to perform brainstem parcellation [@Iglesias2015BayesianMRI], hippocampal subfields segmentation [@Iglesias2015AMRI],
+  and / or probabilistic atlas-based segmentation of the thalamic nuclei [@Najdenovska2018In-vivoImaging].
+All segmented structures are combined at the end of process to create the final parcellation nifti image
+  at each scale along with the corresponding label index color mapping file in accordance to the BIDS Derivatives
+  specifications.
+
+\textbf{A graphical user interface reflecting the workflow structure.}
+CMP3 takes advantage of Traits/TraitsUI framework
+  (\href{http://docs.enthought.com/traits/}{http://docs.enthought.com/traits/}) for building an
+  interactive Graphical User Interface (GUI), to give to pipelines and stages a graphical representation,
+  which is easy to understand and extend.
+This has enabled the design of a GUI aka `cmpbidsappmanager` (\autoref{fig:gui}) that reflects
+  the modular structure of the processing workflow.
+It has been designed to guide and support the user in all the steps required to
+  perform an analysis (\autoref{fig:gui}).
+
+![\textbf{Graphical User Interface of the Connectome Mapper 3.}
+A typical procedure to perform an analysis would consists of
+  (a)   the selection of the root directory of the BIDS dataset to be analyzed,
+  (b)   the creation/edition of the different pipeline configuration files,
+  (c)   the configuration of the BIDS App run and its execution, and
+  (d)   the inspection of stage outputs with fsleyes, freeview, mrview, or TrackVis
+        depending on the tool involved in the stage.
+\label{fig:gui}](cmp3-gui-paper.png)
+
+Each pipeline can be individually configured and executed with the aid of the user-friendly
+  GUI and the output of each stage can be visually reviewed, enabling the user to keep
+  an eye on the data being processed and easily understand the cause of the problems, change the
+  parameters and re-execute when results at a given stage are found not to be satisfactory.
+In this way, CMP3 simplifies the creation of connectomes and makes it a straightforward process
+  even for users not familiar with Nipype and software container technology.
+Nevertheless, it still fulfils the needs of advanced users in charge of analyzing huge amount of data,
```

Comment:
Typo:
analyzing huge amount of data -> analyzing huge amounts of data

OR

analyzing huge amount of data -> analyzing a huge amount of data